### DISCUSSION TITLE
Request for Reference Script for Integrating IntelOwl with Wazuh

### DISCUSSION THREAD

**Discussion by Ghiyarandhika** [intelowlproject/IntelOwl#2482] [Q&A]
Hello,

I am currently working on integrating IntelOwl with Wazuh to enhance the threat intelligence capabilities within my environment. I’ve been searching for reference scripts or guidance on how to effectively set up this integration, but I haven’t found any comprehensive examples.

Could you please provide any reference scripts, documentation, or pointers that would help in integrating IntelOwl with Wazuh? Specifically, I am interested in:

How to forward alerts from Wazuh to IntelOwl for analysis.
Configuration steps required in both Wazuh and IntelOwl for seamless integration.
Any best practices or recommendations for optimizing this setup.
Any help or guidance you can provide would be greatly appreciated.

Thank you!

**Comment by mlodic** [comment]
hey,

here we can provide only community support. Community support implies maintaining our documentation, helping in common problems, bug fixing and so on.

Your request is a consultancy request which is pretty complex and would need to consider a lot of stuff. If you are interested we can provide paid consultancy out of a donation to the project: https://opencollective.com/intelowl-project

**Comment by Ghiyarandhika** [comment]
how much for the paid consultation?

**Reply by mlodic** [reply]
contact us on https://x.com/intel_owl
